packages/core/src/devtools/isDevToolsEnabled.ts

```diff
@@ -0,0 +1,13 @@
+const isEnabled = () => {
+  let griffelDevtoolsEnabled = false;
+  if (process.env.NODE_ENV !== 'production') {
+    try {
+      griffelDevtoolsEnabled = !!window.localStorage['griffelDebug'];
```

Comment:
I am not entirely sure that we will use `localStorage` for this. Is there sense for now to export `false`?

```ts
export const isDevToolsEnabled = false
```

And once we will have an extension we can use the real condition.